Dev_env.md

```diff
@@ -0,0 +1,10 @@
+## Dev env
+In case you would like to set a dev environment for the testing purposes,  
+you could use a great role written by chrismeyersfsu.  
+The provision_docker [1] role creates a container and executes the playbook on it using the docker connection driver.  
+See the role readme for more details.
+
+Use the following Dockerfiles [2] for the tests.  
+
+[1] - https://github.com/chrismeyersfsu/provision_docker  
```

Comment:
Thanks.
It will be great if you can add a Fedora image.
Also it makes sense to add TravisCI.

Reply:
I will definitely add TravisCI to the role once finish all the todo list.
See at the bottom of the README.

Regarding the Fedora, I not sure.
Here is the link to the supported distributions and versions by Bareos.
http://download.bareos.org/bareos/release/latest/

Supported Fedora versions are old - 23, 24.
I'm not sure it makes sense to add such an old version support.

Reply:
As discussed offline, I will add Fedora images to the playbook.